server/src/test/java/org/eclipse/openvsx/UserAPITest.java

```diff
@@ -542,6 +545,16 @@ private String errorJson(String message) throws JsonProcessingException {
     
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+        
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.

Reply:
Just tested this. Please add SimpleMeterRegistry.class to the @MockBean list and remove the UrlConfigService and MeterRegistry `@Bean` methods.